analysis_schema/save_schema_file.py

```diff
@@ -3,9 +3,17 @@
 # This code will save an empty (no values have been entered) model to a json file
 # which will be referenced by the user
 
-analysis_model_schema = ytModel(Data={"FileName": ""}, Plot=[{}])
 
-with open("yt_analysis_schema.json", "w") as file:
-    file.write(analysis_model_schema.schema_json(indent=2))
+def save_schema():
+    """
+    A function to create a schema file
+    """
 
-print("Schema is has been saved!")
+    analysis_model_schema = ytModel(
+        Data=[{"DatasetName": "", "FileName": ""}], Plot=[{}]
+    )
+
+    with open("../analysis_schema/yt_analysis_schema.json", "w") as file:
```

Comment:
more of a comment for later -- but might want a `filename` argument in this function -- it could default to the path you have here, but as it is the code assumes that you're working in the github repo. Since this is mostly for updating the schema after changes to the code, that's OK for now.

Reply:
yeah, I wasn't sure if I should add an arg or default filename and dataset name, since that code is an empty model used to create the schema. I'm open to any suggestions on this.

Reply:
ya, nothing to do for now... but ultimately it relates to how we actually distribute the actual schema file.